docs/api.md

```diff
@@ -1,18 +1,3 @@
-<!-- START doctoc generated TOC please keep comment here to allow auto update -->
```

Comment:
Looks like you don't have doctoc installed: https://github.com/bazelbuild/rules_jvm_external#generating-documentation

Reply:
oops, didn't notice this in the diff